### ISSUE TITLE
Safari desktop va-radio keyboard navigation doesn't work

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#1838]
Click on radio, then press Tab,

expecting to select next radio

really selects Search

https://vuestic.dev/en/ui-elements/radio
